.vscode/launch.json

```diff
@@ -33,6 +33,18 @@
                 "${workspaceFolder}/examples/templates/render/hello_jupiter",
                 "example_test"
             ]
-        }
+        },
+
+        {
+            "name": "Debug abc upgrade",
+            "type": "go",
+            "request": "launch",
+            "mode": "debug",
+            "program": "cmd/abc/abc.go",
+            "args": [
+                "upgrade",
+                "/usr/local/google/home/revell/git/abcxyz/infra-gcp"
```

Comment:
This is not common for use?

Reply:
It's a good starting point for other users to use, if they wish.